src/deploy.ts

```diff
@@ -145,7 +145,7 @@ class Deployer {
       if (this.effects.clack.isCancel(choice) || !choice)
         throw new CliError("User canceled deploy", {print: false, exitCode: 0});
 
-      ({currentUser, apiKey} = await loginInner(this.effects));
+      ({currentUser, apiKey} = await loginInner(this.effects, {pollTime: this.deployOptions.deployPollInterval}));
```

Comment:
Happy to take this change, but I'm pretty sure it is an error that the test is even hitting this line. I'll look into it more.